ledger/acctupdates.go

```diff
@@ -866,23 +877,41 @@ func (au *accountUpdates) accountsInitialize(ctx context.Context, tx *sql.Tx) (b
 				}
 			}
 
-			// this trie Evict will commit using the current transaction.
-			// if anything goes wrong, it will still get rolled back.
-			_, err = trie.Evict(true)
-			if err != nil {
-				return 0, fmt.Errorf("accountsInitialize was unable to commit changes to trie: %v", err)
+			if pendingAccounts >= trieRebuildCommitFrequency {
+				// this trie Evict will commit using the current transaction.
+				// if anything goes wrong, it will still get rolled back.
+				_, err = trie.Evict(true)
+				if err != nil {
+					return 0, fmt.Errorf("accountsInitialize was unable to commit changes to trie: %v", err)
+				}
+				pendingAccounts = 0
```

Comment:
Q: how is it related to the perf fix of `encodedAccountsRange`?

Reply:
separating the Evict frequency from the chunk size decouple the database reading chunk from the trie commit/evict calls. Calling Evict too frequently causes us to drop pages that are needed further on. We don't want to skip the Evict, since the host system might not have enough memory to process the entire trie. That separation was giving us about 40% performance increase.